static/js/util/withTracker.js

```diff
@@ -14,9 +14,30 @@ const withTracker = (WrappedComponent: Class<React.Component<*, *>>) => {
     ReactGA.initialize(SETTINGS.gaTrackingID, { debug: debug, testMode: test })
   }
 
+  if (SETTINGS.gaGTrackingID) {
+    const url = `https://www.googletagmanager.com/gtag/js?id=${
+      SETTINGS.gaGTrackingID
+    }`
+    const gaScript = document.createElement("script")
```

Comment:
This appends the gtag.js snippet to the header https://developers.google.com/analytics/devguides/collection/gtagjs#install_the_global_site_tag